sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -415,6 +477,7 @@ internal class RadarApiClient(
         val uri = Uri.parse(host).buildUpon()
             .appendEncodedPath("v1/trips/")
             .appendEncodedPath(externalId)
+            .appendEncodedPath("update")
```

Comment:
@corypisano Is the API endpoint now `/v1/trips/:externalId/update`? Trying to stay RESTful, doesn't the `PATCH` imply an update?

Reply:
Don't love that the new URL isn't RESTful, but if we feel like we have to do it, that's fine. cc @matthewisabel @palmakkar @tjulien

Reply:
the issue is the old PATCH endpoint wasn't really restful, was overloaded with create & update. so either need a V2 or new endpoint for keeping it as just a true PATCH going forward

Reply:
As long as we're being deliberate/thoughtful here I defer to you all on final naming